clientv3/snapshot/v3_snapshot.go

```diff
@@ -183,18 +183,26 @@ func (s *v3Manager) Status(dbPath string) (ds Status, err error) {
 			if b == nil {
 				return fmt.Errorf("cannot get hash of bucket %s", string(next))
 			}
-			h.Write(next)
+			if _, err := h.Write(next); err != nil {
+				return fmt.Errorf("cannot write bucket %s : %v", string(next), err)
+			}
 			iskeyb := (string(next) == "key")
-			b.ForEach(func(k, v []byte) error {
-				h.Write(k)
-				h.Write(v)
+			if err := b.ForEach(func(k, v []byte) error {
+				if _, err := h.Write(k); err != nil {
+					return err
```

Comment:
wrap the error with "cannot write to bucket %s"?

Reply:
@xiang90 
Your comment has been addressed.